<ui version="4.0" >
 <class>EditPlayersDialog</class>
 <widget class="QDialog" name="EditPlayersDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QTreeView" name="playerTreeView" >
       <property name="uniformRowHeights" >
        <bool>true</bool>
       </property>
       <property name="animated" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>178</width>
           <height>24</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton" >
         <property name="text" >
          <string>ok</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QToolButton" name="addButton" >
       <property name="text" >
        <string>add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeButton" >
       <property name="text" >
        <string>remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="editButton" >
       <property name="text" >
        <string>edit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>138</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
